nova_vm/src/ecmascript/abstract_operations/type_conversion.rs

```diff
@@ -140,10 +140,10 @@ pub(crate) fn ordinary_to_primitive(
 }
 
 /// ### [7.1.2 ToBoolean ( argument )](https://tc39.es/ecma262/#sec-toboolean)
-pub(crate) fn to_boolean(agent: &mut Agent, argument: Value) -> JsResult<Value> {
+pub(crate) fn to_boolean(agent: &mut Agent, argument: Value) -> JsResult<bool> {
```

Comment:
**suggestion:** Also get rid of the JsResult since we're not using it. The spec also probably makes no mention of a throw completion so it's perfectly within spec to leave it out.